vendor/validator/src/type_validators.js

```diff
@@ -194,7 +194,7 @@ function validateArray(schema, data, path, parent, prop, recurse) {
 
 	// minItems and maxItems
 	// console.warn('Running array validation with min items: ', schema['minItems'], data.length)
-	if (typeof schema['minItems'] === 'number' && data.length < schema['minItems']) this.addError(path, schema, `At least ${ schema['minItems'] } record(s) are required`)
+	if (typeof schema['minItems'] === 'number' && data.length < schema['minItems']) this.addError(path, schema, `At least ${ schema['minItems'] === 1 ? 'one' : schema['minItems'] } record${schema['minItems'] > 1 ? '(s)' : ''} are required`)
```

Comment:
record(s) -> records
one record are -> one record is
maybe something like:
```suggestion
	if (typeof schema['minItems'] === 'number' && data.length < schema['minItems']) this.addError(path, schema, `At least ${ schema['minItems'] === 1 ? 'one' : schema['minItems'] } record${schema['minItems'] > 1 ? 's are' : ' is'} required`)
```